generators/server/templates/src/main/java/package/web/rest/dto/_UserDTO.java

```diff
@@ -13,7 +13,7 @@
  */
 public class UserDTO {
 
-    public static final int PASSWORD_MIN_LENGTH = 5;
+    public static final int PASSWORD_MIN_LENGTH = 4;
```

Comment:
I changed this from 5 to 4 because one of our default passwords ("user") is only 4 characters.